docs/dev-local-node.md

```diff
@@ -132,8 +135,7 @@ For more information on the script tool disco parrot (dp) [disco parrot](https:/
 
 Here are misc. script to do operations on the local node and to change time. More scripts will be created as they are needed, enivronment variables can be used or passed into the command like ETHEREUM_PRIVATE_KEY is in the examples below:
 
-    ** change time
-    ETHEREUM_PRIVATE_KEY=fae42052f82bed612a724fec3632f325f377120592c75bb78adfcceae6470c5a node scripts/helpers/change-time.js <unix timestamp>
+For changing time on the contracts, use flash in augur.js:
 
     ** create open order
```

Comment:
Should these three non-flash script items become a separate code block?

Reply:
I agree, I'll move down with more recent changes.